.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
We already have a .eslintrc file in this repo, check which is used and remove the other :)